ansible/playbooks/monitoring.yml

```diff
@@ -188,6 +188,12 @@
               labels:
                 dispatch: pagerduty-cratesio
 
+            - alert: TooManyQueuedJobs
+              expr: cratesio_service_background_jobs{env="prod"} > 100
```

Comment:
we introduced a `priority` column for background jobs lately and when we backfilled the `rust-version` values a few weeks ago we had several thousand low-priority background jobs queued up. I guess such a case would trigger the pagerduty alert if we merged it like this. Maybe we should somehow feed the priority into the monitoring system too? For the `bin/monitor` application we only consider `priority >= 0` (normal or higher).

Reply:
I don't think the priority is exposed through the `cratesio_service_background_jobs` metric though. We should add it first.

Reply:
Will update once https://github.com/rust-lang/crates.io/pull/6871 is deployed.

Reply:
the PR is deployed in staging and production now